apps/api/src/admin/applicant_review_processor.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from models.ApplicationData import Decision
+
+
+def include_hacker_app_fields(
+    applicant_record: dict[str, Any], accept_threshold: float, waitlist_threshold: float
+) -> None:
+    _include_decision_based_on_threshold(
+        applicant_record, accept_threshold, waitlist_threshold
+    )
+    _include_num_reviewers(applicant_record)
+    _include_avg_score(applicant_record)
+    _omit_reviews(applicant_record)
```

Comment:
Thought: since the resulting record is validated through the type adapter for HackerApplicantSummary, I don't think it's necessary to omit the reviews field since the model validator will ignore it.